gobblin-metastore/src/main/resources/gobblin_job_history_store.sql

```diff
@@ -91,29 +91,29 @@ CREATE TABLE IF NOT EXISTS gobblin_task_metrics (
 );
 
 CREATE TABLE IF NOT EXISTS gobblin_job_properties (
-  property_id BIGINT(21) NOT NULL AUTO_INCREMENT,
-  job_id VARCHAR(128) NOT NULL,
-  property_key VARCHAR(128) NOT NULL,
-  property_value TEXT NOT NULL,
+	property_id BIGINT(21) NOT NULL AUTO_INCREMENT,
+	job_id VARCHAR(128) NOT NULL,
+	property_key VARCHAR(128) NOT NULL,
+	property_value MEDIUMTEXT NOT NULL,
```

Comment:
de-dups will help. It will also help if we stop adding all global state props to each `WorkUnitState`. I opened a ticket for addressing this: https://github.com/linkedin/gobblin/issues/696